build_functions/CMakeLists.txt

```diff
@@ -160,6 +160,8 @@ macro(set_default_build_options)
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /fsanitize=address")
     endif()
 
+    configure_file("${build_c_tests_internal_dir}/ignore_function_names_vld.ini" "$ENV{ProgramFiles\(x86\)}/Visual Leak Detector/vld.ini" COPYONLY)
```

Comment:
> configure_file("${build_c_tests_internal_dir}/ignore_function_names_vld.ini" "$ENV{ProgramFiles\(x86\)}/Visual Leak Detector/vld.ini" COPYONLY) [](http://example.com/codeflow?start=3&length=144)

Shouldn't this just come with the new vld version?
I think we should not mangle existing versions (copying to program files) with our SW.